My-Spec-Tutorial.package/RadioButtonGroup.extension/instance/selectedButton.st

```diff
@@ -0,0 +1,4 @@
+*My-Spec-Tutorial
+selectedButton
+
+	^ buttons detect: [ :b | b state ] ifNone: [ nil ]
```

Comment:
thank you very much for your time. Yours suggestion helped me a lot.
i am facing one problem, i.e., i am trying clone your repo of hello
world. After
that i tried saving and opening that package using Monticello Browser. But
whenever i try save & open that package in pharo, all class and extension
files are getting deleted and pharo is showing nothing in that package. So
i am looking into that problem.
Actually i added all the suggestion you suggested manually in my package
file but even though i am facing a problem in last suggestion, i.e., making
a RadioButtonGroup.extension.
I am working on this.

Regards
Rohit


On Sun, Mar 8, 2015 at 12:23 AM, Sean DeNigris <notifications@github.com>
wrote:

> In
> My-Spec-Tutorial.package/RadioButtonGroup.extension/instance/selectedButton.st
> <https://github.com/rohitsukhwal/HelloWorld/pull/1#discussion_r25998419>:
>
> > @@ -0,0 +1,4 @@
> > +*My-Spec-Tutorial
> > +selectedButton
> > +
> > +	^ buttons detect: [ :b | b state ] ifNone: [ nil ]
>
> See how much simpler this is than that switch statement! One could easily
> argue that this should have been provided in the kernel, and if you open an
> issue, it is a good candidate for integration.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/rohitsukhwal/HelloWorld/pull/1/files#r25998419>.
>